docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -67,57 +63,149 @@ The package is published in this location:
 
 ## Initialization
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the UID2 service. It also allows you to configure retry intervals in the event of errors.
+- DSPs should create an instance of the `BidstreamClient` class.
+- Sharers should create an instance of the `SharingClient` class.
+- Publishers should create an instance of the `Uid2PublisherClient` class.
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the UID2 service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to UID2, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+You will need to provide the values necessary for the SDK to authenticate with the UID2 service.
+
+| Parameter    | Description                                                                               |
+|:-------------|:------------------------------------------------------------------------------------------|
+| `endpoint`   | The endpoint for the UID2 service. See [Environments](../getting-started/gs-environments) |
+| `auth_key`   | The API key. See [UID2 Credentials](../getting-started/gs-credentials).                   |
+| `secret_key` | The client secret. See [UID2 Credentials](../getting-started/gs-credentials).             |
 
 ## Interface 
 
-The interface allows you to decrypt UID2 advertising tokens and return the corresponding raw UID2. 
+The `BidstreamClient` class allows you to decrypt UID2 tokens into raw UID2s.
+For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
+
+The `SharingClient` class allows you to encrypt raw UID2s into UID2 tokens and decrypt UID2 tokens into raw UID2s.
+
 
 >NOTE: When you use an SDK, you do not need to store or manage decryption keys.
 
-If you're a DSP, for bidding, call the interface to decrypt a UID2 advertising token and return the UID2. For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
+### Encryption Response Content
+
+When encrypting with the `SharingClient`, the SDK returns the following information:
```

Comment:
When encrypting with the `SharingClient`, the SDK returns the information shown in the following table.
(style guidelines, we don't use a colon before a table, only before steps or bullets).